lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -398,53 +398,52 @@ define((require, exports, module) ->
       }
 
 
-  exports.ContextHandler = ContextHandler
-  exports.ConstrainedTokenIterator = ConstrainedTokenIterator
-  exports.EquationRangeHandler = EquationRangeHandler
-  exports.setupPreviewer = (editor, popoverHandler, katexLoader, I18N) ->
-    myKatexLoader = katexLoader
-
-    jqEditorContainer = $(editor.container)
-    KATEX_OPTIONS = { displayMode: true, throwOnError: false }
+  contextHandler = null
```

Comment:
pls rename it to `myContextHandler` to indicate it is not a local variable (we use this naming convention in the main Papeeria code)